### ISSUE TITLE
Add a buffer for min_value before attempting send

### ISSUE THREAD

**Issue by tomreece** [tomreece/pucauto#67]
Currently Pucauto accepts a bundle if the recipient has `points >= min_value`.

Make it `points >= min_value + BUFFER`.

**Comment by eengstrom** [comment]
Would this be an attempt to ensure that the recipient's points aren't all used up before Pucauto gets to send all of our matching cards?        I'm not sure I would use that feature, so can the `buffer` be a config option?

**Comment by tomreece** [comment]
@eengstrom Yes, the goal is to ensure min_value is met in case some cards are grabbed by other users. I'm not sure what buffer value should be, but it will be configurable and optional.
